skimage/feature/tests/test_match.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-from skimage._shared.testing import assert_equal
+from skimage._shared.testing import assert_equal, parametrize
```

Comment:
I'm missing why this edit is necessary... :/

Reply:
sorry. that was left over from when I was thinking of using parametrize for the tests.